consoleproxy/consoleproxy/plugins/vnc.py

```
@@ -0,0 +1,203 @@
+
+    def _get_pid_on_port(self, port):
+        out = shell.ShellCmd('netstat -anp | grep ":%s" | grep LISTEN' % port)
+        out(False)
+        out = out.stdout.strip()
+        if "" == out:
+            return None
+
+        pid = out.split()[-1].split('/')[0]
+        try:
+            pid = int(pid)
+            return pid
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)